pkgs/development/compilers/llvm/common/libunwind/default.nix

```diff
@@ -71,10 +71,10 @@ stdenv.mkDerivation (rec {
     ninja python3
   ];
 
-  cmakeFlags = lib.optional (lib.versionAtLeast release_version "15") "-DLLVM_ENABLE_RUNTIMES=libunwind"
-    ++ lib.optional (!enableShared) "-DLIBUNWIND_ENABLE_SHARED=OFF"
+  cmakeFlags = lib.optional (lib.versionAtLeast release_version "15") (lib.cmakeFeature "LLVM_ENABLE_RUNTIMES" "libunwind")
+    ++ [ (lib.cmakeBool "LIBUNWIND_ENABLE_SHARED" enableShared) ]
```

Comment:
Shouldn't this go first, since it's unconditionally included?

Reply:
Yeah, that's not a bad idea.